src/AutoRest.CSharp/Common/Generation/Writers/RequestWriterHelpers.cs

```diff
@@ -15,6 +15,7 @@
 using Azure.Core;
 using AutoRest.CSharp.Utilities;
 using Azure;
+using AutoRest.CSharp.Common.Output.Models;
```

Comment:
Nit: Why is this needed? nothing changed in this file.

Reply:
removed